examples/nucleo_f072rb/independend_watchdog/main.cpp

```diff
@@ -25,7 +26,7 @@ main()
 	Board::initialize();
 	LedD13::setOutput();
 	// set the watchdog timeout to 4 seconds
-	Iwdg::initialize(Iwdg::Prescaler::Div32, 0x0FFFu);
+	Iwdg::initialize<SystemClock, 4.0>();
```

Comment:
Sadly `std::chrono::microseconds` are not allowed as a template type.

Reply:
If you really wanted to, you could define a custom microsecond struct with a non-explicit converting constructor that takes a `std::chrono::microseconds` parameter. 
That could be used as the template argument type and `std::chrono::microseconds` would implicitly convert to it. A bit ugly, but it does the job.